DesignPattern-TryCatch/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Exception Handling via Try-Catch pattern using Scope action",
+    "description": "This Logic Apps workflow represents the try-catch pattern for exception handling using Scope action",
+    "tags": [
+        "Exception Handling, try catch"
```

Comment:
I noticed this after merging. Since tags is an array, we want separate strings for all of those (moving forward). I will make the change in addition to other changes we discussed together